port/unix/omrsysinfo.c

```diff
@@ -3150,8 +3182,11 @@ omrsysinfo_os_kernel_info(struct OMRPortLibrary *portLibrary, struct OMROSKernel
 	BOOLEAN success = FALSE;
 
 #if defined(LINUX)
+#if !defined(OMRZTPF)
 	struct utsname name = {0};
-
+#else /* !defined(OMRZTPF) */
+    struct utsname name = {{0}};
```

Comment:
Minor nitpick: the indent on this line doesn't match the line above.  I think it needs to be tabbed in one level deeper.